### ISSUE TITLE
Unable to zoom out on an image

### ISSUE THREAD

**Issue by Artur-** [microsoft/vscode#131080]
Extension version: v0.29.2
VSCode Version: 1.59.0
OS: GitHub editor / mac

Steps to Reproduce:

1. Open https://github.dev/Artur-/screenshot-public/pull/1
2. Click "image.png"
3. Click on the right side image, it zooms in
4. Hold ctrl down and the icon turns into a "zoom out" icon
5. Click

Expected outcome:
Zoom out

Actual outcome: You see a menu with "Cut", "Copy", "Paste"

**Comment by alexr00** [comment]
@mjbvz maybe a safari issue?

**Comment by Artur-** [comment]
In Chrome, ctrl-click only shows the menu
In Safari, ctrl-click zooms out and also shows the menu

**Comment by alexr00** [comment]
Thanks for the additional details. In Edge, I get only the zoom out.

**Comment by hardikkoul** [comment]
Working fine in firefox.

**Comment by san-coding** [comment]
@alexr00 Issue exists in Safari browser, how can I fix this

**Comment by san-coding** [comment]
@alexr00 yes it works in safari , but the the cursor remains zoom out cursor even when I don't press control, it doesn't change back to + symbol

**Comment by Arkadipta-Chatterjee** [comment]
Can i work on this??

**Comment by akshay48-web** [comment]
if you didn't work to zoom in or zoom out you use simply method on keybord key ctrl + or ctrl - .
